c2corg_ui/static/lib/angular-bootstrap/how to

```diff
@@ -0,0 +1,21 @@
+1) go to https://angular-ui.github.io/bootstrap/ and click on "create a custom build"
+
+2) select the necessary modules, in our case those are : 
+	- datepicker
+	- modal
+	- dropdown
+	- popover
+	- tabs
+	- tooltip
+	- rating
+
+3) update the scripts in base.html
```

Comment:
You should also say where the generated builds must be saved (in the current dir?).